### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#423] [Esami & Esoneri M-Z]
## Traccia

<img width="582" alt="image" src="https://github.com/user-attachments/assets/2a002e1b-db2b-473e-a7af-5b420d1c8c82">

> Questo esercizio è stato proposto **anche** come **Esercizio 1** della traccia di esame **del 22 gennaio 2019**

**Comment by rimaout** [comment]
<img width="500" alt="image" src="https://github.com/user-attachments/assets/c21485cd-7e0a-450d-bf89-50bdac0a8c74">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [*20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/a239cb47-631b-41ad-b18c-05403e18842e)

\* _Questo esercizio era in comune rispetto alla traccia del 22 gennaio 2019, gli esercizi 7, 8 e 9 sono specifici delle due rispettive tracce, per quelli vedi [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf)_
